### ISSUE TITLE
Don't reload the last discriminator layer

### ISSUE THREAD

**Issue by jdayton3** [jdayton3/Confounded#69]
This causes issues when there are a different number of batches in the train vs test dataset. Since the primary use of pre-training is probably having different train and test datasets (like a large train set and a small test set), we want to just remove and retrain the final discriminator layer.

**Comment by jdayton3** [comment]
Closed by bd49688dee0db46f4c3dc334ea65786daaf99ba2
